libs/npm/core/src/hooks/summon/useSummon.tsx

```diff
@@ -1,22 +1,32 @@
-import axios from 'axios';
+import axios, { AxiosInstance } from 'axios';
```

Comment:
Fairly significant change here.  Trying to figure out why we get state token values.  I've added global state management to this.  Each unique baseURL will have it's own global axios instance.  Additionally they all use a global `getToken()` function that should have the latest token value.